DockerfileLeanFoundation

```diff
@@ -265,5 +265,13 @@ RUN wget https://cdn.quantconnect.com/h2o/h2o-3.30.0.3.zip && \
 # Remove black-listed packages
 RUN pip uninstall -y s3transfer
 
+# Install a Python Virtual Environment to contain upgraded libraries for users to opt into (Pandas, Tensorflow)
+RUN python -m venv /venv && \
+    . /venv/bin/activate && \
+    pip install --upgrade pip && \
+    pip install \
+    pandas==1.1.5 \
```

Comment:
Should use `pip install --no-cache-dir`, don't think there is a need to upgrade `pip`? Name here is `venv`? Let's give it a more significant name 👍

Reply:
I did need to upgrade pip in order to find TF 2.5.0, but will do the rest 😁